vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -48,8 +43,8 @@ kubectl describe ingress/default-sso-in-namespace-ingress -n entando
 10. Click `Add Selected` to add `superuser` to the `Assigned Roles`. This change will be saved automatically. 
 11. Go to the `Credentials` tab and copy the `Secret` shown there for use in the next section
  
-### Create env.json
-1. Create a directory where you'll run the bundler and go to that directory:
+### Create the `env.json`
```

Comment:
@nshaw, is there a general term for env.json type files? Bundle Environment config file? Environment config file for Bundle?

Reply:
-> Create the `env.json` configuration file